diff --git a/grader/settings.py b/grader/settings.py
index 8300aa9..c11ffa4 100644
--- a/grader/settings.py
+++ b/grader/settings.py
@@ -26,6 +26,7 @@ INSTALLED_APPS = (
     # 'django.contrib.contenttypes',
     # 'django.contrib.sessions',
     # 'django.contrib.messages',
+    'corsheaders',
     'staticfileserver', # override for runserver command, thus this needs to be before django contrib one
     'django.contrib.staticfiles',
     'access',
@@ -38,12 +39,15 @@ MIDDLEWARE = [
     # 'django.middleware.security.SecurityMiddleware',
     # 'django.contrib.sessions.middleware.SessionMiddleware',
     # 'django.middleware.csrf.CsrfViewMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     # 'django.contrib.auth.middleware.AuthenticationMiddleware',
     # 'django.middleware.locale.LocaleMiddleware',
     'django.middleware.common.CommonMiddleware',
 ]
 
+CORS_ALLOW_ALL_ORIGINS = True
+
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
diff --git a/requirements.txt b/requirements.txt
index 091bffc..b412eb8 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -4,3 +4,4 @@ requests~=2.25.1
 PyYAML~=5.4.1
 docutils==0.16
 python-magic~=0.4.22
+django-cors-headers ~= 3.7.0
