diff --git a/grader/settings.py b/grader/settings.py
index 86cf27e..062fed5 100644
--- a/grader/settings.py
+++ b/grader/settings.py
@@ -49,6 +49,7 @@ INSTALLED_APPS = (
     # 'django.contrib.contenttypes',
     # 'django.contrib.sessions',
     # 'django.contrib.messages',
+    'corsheaders',
     'staticfileserver', # override for runserver command, thus this needs to be before django contrib one
     'django.contrib.staticfiles',
     'access',
@@ -59,6 +60,7 @@ MIDDLEWARE = [
     # 'django.middleware.security.SecurityMiddleware',
     # 'django.contrib.sessions.middleware.SessionMiddleware',
     # 'django.middleware.csrf.CsrfViewMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     # 'django.contrib.auth.middleware.AuthenticationMiddleware',
     # 'django.middleware.locale.LocaleMiddleware',
@@ -66,6 +68,8 @@ MIDDLEWARE = [
     'aplus_auth.auth.django.AuthenticationMiddleware',
 ]
 
+CORS_ALLOW_ALL_ORIGINS = True
+
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
diff --git a/requirements.txt b/requirements.txt
index 0996bef..58fe1aa 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -6,3 +6,4 @@ docutils~=0.17.1
 python-magic~=0.4.22
 aplus-auth~=0.1.0
 docker~=5.0.0
+django-cors-headers ~= 3.7.0
